test/ptp/ptp.go

```diff
@@ -102,7 +110,9 @@ var _ = Describe("[ptp]", func() {
 		var masterProfile, slaveProfile string
 
 		execute.BeforeAll(func() {
-
+			if isSingleNode {
+				Skip("Running in single node mode")
```

Comment:
Can you expand the reason of the skipping?
Also, this won't work because beforeAll is a beforeEach that runs only the first time.
The result will be that only the first test will be skipped.
Add a beforeEach that does the skipping

Reply:
This is actually meant to skip the configuration in beforeAll (which would fail for sno).

Reply:
Yes ,but if you skip in beforeAll, it's gonna skip only the first test of the suite, not all of them

Reply:
There is also a skip in beforeEach.
Not sure we can have the Skip directly in Describe

Reply:
Ah, sorry I did not notice it.
Thanks